Source/WebCore/platform/graphics/FontSelectionAlgorithm.h

```diff
@@ -519,11 +509,6 @@ std::optional<FontSelectionSpecifiedCapabilities> FontSelectionSpecifiedCapabili
     return {{ *weight, *width, *slope }};
 }
 
-constexpr bool operator==(const FontSelectionSpecifiedCapabilities& a, const FontSelectionSpecifiedCapabilities& b)
-{
-    return a.tied() == b.tied();
```

Comment:
Maybe we should delete the `tied` function too and just move the code into the `add(Hasher&)` function?